packages/analytics/dashboard-renderer/src/components/TimeseriesChartRenderer.vue

```diff
@@ -36,3 +38,9 @@ const options = computed<AnalyticsChartOptions>(() => ({
   chartDatasetColors: props.chartOptions.chartDatasetColors,
 }))
 </script>
+
+<style scoped lang="scss">
+.analytics-chart {
+  height: v-bind('`${height}px`')
```

Comment:
question: did you verify this is outputting the desired value in the browser dev tools? (for all the computed CSS vals)

Reply:
yes its always coming in as a number, the dashboard renderer enforces setting a width and a height for each tile

Reply:
in this case height is all we need cause width is always gonna be 100% by default